packages/bot-web-ui/src/app/app.jsx

```diff
@@ -16,13 +16,21 @@ import RootStore from 'Stores';
 import GTM from 'Utils/gtm';
 import BotBuilder from 'Components/dashboard/bot-builder';
 import './app.scss';
+import { setColors } from '../../../bot-skeleton/src/scratch/hooks/colours';
+import { useDarkMode } from '../components/dashboard/hooks';
 
 const App = ({ passthrough }) => {
     const { root_store, WS } = passthrough;
     const [is_loading, setIsLoading] = React.useState(true);
     const root_store_instance = React.useRef(new RootStore(root_store, WS, DBot));
     const { app, common, core } = root_store_instance.current;
     const { showDigitalOptionsMaltainvestError } = app;
+    const dbot_theme_mode = useDarkMode(root_store);
+    const { is_dark_mode } = dbot_theme_mode;
+
+    React.useEffect(() => {
```

Comment:
@rupato-deriv As we discussed, You can probably use the `root_store` value directly here in the `useEffect` and we should be able to get rid of all the other changes.

Although I would suggest installing the `@deriv/stores` package and then using the `useStore` hook to get the value, You can ignore the suggestion 🙇🏻